### PR TITLE
UI: New Crowdin updates

### PR DESCRIPTION
**Pull request by AcK77**
> ~~• Chinese Simplified -> 100%~~
> ~~• Chinese Traditional -> 100%~~
> ~~• French -> 100%~~
> ~~• German -> 100%~~
> • Greek -> 94%
> • Hebrew -> 98%
> ~~• Italian -> 100%~~
> ~~• Japanese -> 100%~~
> ~~• Korean -> 100%~~
> • Polish -> 98%
> • Portuguese, Brazilian -> 94%
> ~~• Russian -> 100%~~
> ~~• Spanish -> 100%~~
> ~~• Turkish -> 100%~~
> • Ukrainian -> 88%

If you want to help to translate the project, https://crowdin.com/project/ryujinx/

### CODE REVIEW

**Review by gdkchan — APPROVED** [review]

**Comment by Narugakuruga** [comment]
There's something happened in Chinese simplified.

I made a full 100% translation already. But now 6% lines are accidentally deleted by the website. Maybe something went really off in the process?

**Comment by AcK77** [comment]
As I said on our Discord, sadly crowdin hadn’t like our files move to `src` folder, I was able to restore some of locale files but there are some rollback on others. Which is bad for us but we can’t do anything more except retranslate missing locales.
Sorry for the inconvenience.

**Comment by TSRBerry** [comment]
@AcK77 could we merge this?

**Comment by AcK77** [comment]
I updated the languages state, a bunch of them aren't 100% and it could be nice to get a bit more or them completed before merging this PR.

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 1]
```diff
@@ -26,6 +27,9 @@
   "MenuBarToolsInstallFirmware": "安裝韌體",
   "MenuBarFileToolsInstallFirmwareFromFile": "從 XCI 或 ZIP 安裝韌體",
   "MenuBarFileToolsInstallFirmwareFromDirectory": "從資料夾安裝韌體",
+  "MenuBarToolsManageFileTypes": "Manage file types",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:30]: 管理檔案類型

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 2]
```diff
@@ -26,6 +27,9 @@
...
+  "MenuBarToolsInstallFileTypes": "Install file types",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:31]: 安裝檔案類型

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 3]
```diff
@@ -26,6 +27,9 @@
...
+  "MenuBarToolsUninstallFileTypes": "Uninstall file types",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:32]: 卸載檔案類型

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 4]
```diff
@@ -76,7 +80,10 @@
   "SettingsTabGeneralEnableDiscordRichPresence": "啟用 Discord 動態狀態展示",
   "SettingsTabGeneralCheckUpdatesOnLaunch": "自動檢查更新",
   "SettingsTabGeneralShowConfirmExitDialog": "顯示「確認離開」對話框",
+  "SettingsTabGeneralHideCursor": "Hide Cursor:",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:83]: 隱藏滑鼠

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 5]
```diff
@@ -76,7 +80,10 @@
...
+  "SettingsTabGeneralHideCursorNever": "Never",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:84]: 從不

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 6]
```diff
@@ -76,7 +80,10 @@
...
   "SettingsTabGeneralHideCursorOnIdle": "自動隱藏滑鼠",
+  "SettingsTabGeneralHideCursorAlways": "Always",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:86]: 總是

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 7]
```diff
@@ -158,6 +165,7 @@
   "SettingsTabLoggingEnableFsAccessLogs": "記錄檔案存取",
   "SettingsTabLoggingFsGlobalAccessLogMode": "記錄全域檔案存取模式:",
   "SettingsTabLoggingDeveloperOptions": "開發者選項 (警告: 會降低效能)",
+  "SettingsTabLoggingDeveloperOptionsNote": "WARNING: Will reduce performance",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:168]: 警告: 會降低效能

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 8]
```diff
@@ -262,6 +270,7 @@
   "UserProfilesAddNewProfile": "建立帳號",
   "UserProfilesDelete": "刪除",
   "UserProfilesClose": "關閉",
+  "ProfileNameSelectionWatermark": "Choose a nickname",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:273]: 選擇暱稱

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 9]
```diff
@@ -337,6 +346,10 @@
   "DialogFirmwareInstallEmbeddedSuccessMessage": "未找到已安裝的韌體，但 Ryujinx 可以從現有的遊戲安裝韌體{0}.\\n模擬器現在可以執行。",
   "DialogFirmwareNoFirmwareInstalledMessage": "未安裝韌體",
   "DialogFirmwareInstalledMessage": "已安裝韌體{0}",
+  "DialogInstallFileTypesSuccessMessage": "Successfully installed file types!",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:349]: 安裝檔案類型成功！

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 10]
```diff
@@ -337,6 +346,10 @@
...
+  "DialogInstallFileTypesErrorMessage": "Failed to install file types.",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:350]: 安裝檔案類型失敗。

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 11]
```diff
@@ -337,6 +346,10 @@
...
+  "DialogUninstallFileTypesSuccessMessage": "Successfully uninstalled file types!",
+  "DialogUninstallFileTypesErrorMessage": "Failed to uninstall file types.",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:352]: 卸載檔案類型失敗。

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 12]
```diff
@@ -337,6 +346,10 @@
   "DialogFirmwareInstallEmbeddedSuccessMessage": "未找到已安裝的韌體，但 Ryujinx 可以從現有的遊戲安裝韌體{0}.\\n模擬器現在可以執行。",
   "DialogFirmwareNoFirmwareInstalledMessage": "未安裝韌體",
   "DialogFirmwareInstalledMessage": "已安裝韌體{0}",
+  "DialogInstallFileTypesSuccessMessage": "Successfully installed file types!",
+  "DialogInstallFileTypesErrorMessage": "Failed to install file types.",
+  "DialogUninstallFileTypesSuccessMessage": "Successfully uninstalled file types!",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:351]: 卸載檔案類型成功！

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 13]
```diff
@@ -368,6 +381,9 @@
   "DialogFirmwareInstallerFirmwareInstallSuccessMessage": "成功安裝系統版本{0}。",
   "DialogUserProfileDeletionWarningMessage": "刪除後將沒有可選擇的使用者帳號",
   "DialogUserProfileDeletionConfirmMessage": "是否刪除選擇的帳號",
+  "DialogUserProfileUnsavedChangesTitle": "Warning - Unsaved Changes",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:384]: 警告 - 未保存的更改

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 14]
```diff
@@ -368,6 +381,9 @@
...
+  "DialogUserProfileUnsavedChangesMessage": "You have made changes to this user profile that have not been saved.",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:385]: “您對此帳號所做的更改尚未保存。

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 15]
```diff
@@ -368,6 +381,9 @@
...
+  "DialogUserProfileUnsavedChangesSubMessage": "Do you want to discard your changes?",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:386]: 是否要放棄您的更改？

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 16]
```diff
@@ -416,6 +432,7 @@
   "DlcManagerEnableAllButton": "啟用全部",
   "DlcManagerDisableAllButton": "停用全部",
   "MenuBarOptionsChangeLanguage": "變更語言",
+  "MenuBarShowFileTypes": "Show File Types",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:435]: 顯示檔案類型

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 17]
```diff
@@ -445,6 +462,7 @@
   "MemoryManagerSoftwareTooltip": "使用軟體記憶體頁管理，最精確但是速度最慢",
   "MemoryManagerHostTooltip": "直接映射記憶體頁到電腦記憶體， JIT 效率高",
   "MemoryManagerUnsafeTooltip": "直接映射記憶體頁，但是不檢查記憶體溢出，JIT效率最高。\nRyujinx 可以存取任何位置的記憶體，因而相對不安全。此模式下只應執行您信任的遊戲或軟體(即官方遊戲)",
+  "UseHypervisorTooltip": "Use Hypervisor instead of JIT. Greatly improves performance when available, but can be unstable in its current state.",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:465]: 使用 Hypervisor 而不是 JIT。 可用時大大提高效能，但在當前狀態下可能不穩定。

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 18]
```diff
@@ -610,5 +628,21 @@
   "Search": "搜尋",
   "UserProfilesRecoverLostAccounts": "恢復遺失的帳號",
   "Recover": "恢復",
-  "UserProfilesRecoverHeading": "在以下帳號找到了一些遊戲存檔"
+  "UserProfilesRecoverHeading": "在以下帳號找到了一些遊戲存檔",
+  "UserProfilesRecoverEmptyList": "No profiles to recover",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:632]: 沒有要恢復的遊戲存檔

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 19]
```diff
@@ -610,5 +628,21 @@
...
+  "GraphicsAATooltip": "Applies anti-aliasing to the game render",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:633]: 對遊戲渲染應用抗鋸齒

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 20]
```diff
@@ -610,5 +628,21 @@
...
+  "GraphicsAALabel": "Anti-Aliasing:",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:634]: 抗鋸齒：

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 21]
```diff
@@ -610,5 +628,21 @@
...
+  "GraphicsScalingFilterLabel": "Scaling Filter:",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:635]: 縮放過濾：

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 22]
```diff
@@ -610,5 +628,21 @@
...
+  "GraphicsScalingFilterTooltip": "Enables Framebuffer Scaling",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:636]: 啟用幀緩衝區縮放

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 23]
```diff
@@ -610,5 +628,21 @@
...
+  "GraphicsScalingFilterLevelLabel": "Level",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:637]: 等級

Comment (chen3feng) [reply src/Ryujinx.Ava/Assets/Locales/zh_TW.json:637]: 設置縮放過濾等級

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 24]
```diff
@@ -610,5 +628,21 @@
...
+  "GraphicsScalingFilterLevelTooltip": "Set Scaling Filter Level",
+  "SmaaLow": "SMAA Low",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:639]: SMAA 低

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 25]
```diff
@@ -610,5 +628,21 @@
...
+  "SmaaMedium": "SMAA Medium",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:640]: SMAA 中

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 26]
```diff
@@ -610,5 +628,21 @@
...
+  "SmaaHigh": "SMAA High",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:641]: SMAA 高

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 27]
```diff
@@ -610,5 +628,21 @@
...
+  "SmaaUltra": "SMAA Ultra",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:642]: SMAA 超高

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 28]
```diff
@@ -610,5 +628,21 @@
...
+  "UserEditorTitle": "Edit User",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:643]: 編輯用戶

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 29]
```diff
@@ -610,5 +628,21 @@
...
+  "UserEditorTitleCreate": "Create User",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:644]: 創建用戶

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 30]
```diff
@@ -610,5 +628,21 @@
...
+  "SettingsTabNetworkInterface": "Network Interface:",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:645]: 網路介面：

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 31]
```diff
@@ -610,5 +628,21 @@
...
+  "NetworkInterfaceTooltip": "The network interface used for LAN features",
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:646]: 用於 LAN 功能的網路介面

File: src/Ryujinx.Ava/Assets/Locales/zh_TW.json [hunk 32]
```diff
@@ -610,5 +628,21 @@
...
+  "NetworkInterfaceDefault": "Default"
```
Comment (chen3feng) [inline src/Ryujinx.Ava/Assets/Locales/zh_TW.json:647]: 默認

**Review by chen3feng — COMMENTED** [review]
Provide CHS & CHT translation, hope helpful!

**Comment by CoolFrenchPerson** [comment]
This is still marked as "Pro Controller" when it should be "Manette Pro"
I couldn't find anything regarding "Pro Controller" in the fr_FR file, so i'll just say it here

image

File: src/Ryujinx.Ava/Assets/Locales/fr_FR.json [hunk 33]
```diff
@@ -337,6 +346,10 @@
   "DialogFirmwareInstallEmbeddedSuccessMessage": "Aucun firmware installé n'a été trouvé mais Ryujinx a pu installer le firmware {0} à partir du jeu fourni.\\nL'émulateur va maintenant démarrer.",
   "DialogFirmwareNoFirmwareInstalledMessage": "Aucun Firmware installé",
   "DialogFirmwareInstalledMessage": "Le firmware {0} a été installé",
+  "DialogInstallFileTypesSuccessMessage": "Types de fichiers installés avec succès!",
+  "DialogInstallFileTypesErrorMessage": "Échec de l'installation des types de fichiers.",
+  "DialogUninstallFileTypesSuccessMessage": "Types de fichiers désinstallés avec succès!",
+  "DialogUninstallFileTypesErrorMessage": "Échec de la désinstallation des types de fichiers.",
   "DialogOpenSettingsWindowLabel": "Ouvrir la fenêtre de configuration",
   "DialogControllerAppletTitle": "Controller Applet",
```
Comment (CoolFrenchPerson) [inline src/Ryujinx.Ava/Assets/Locales/fr_FR.json:354]: As weird as this sounds, this should be "Appliquette de la Manette"

File: src/Ryujinx.Ava/Assets/Locales/fr_FR.json [hunk 34]
```diff
@@ -76,7 +80,10 @@
   "SettingsTabGeneralEnableDiscordRichPresence": "Active Discord Rich Presence",
```
Comment (CoolFrenchPerson) [inline src/Ryujinx.Ava/Assets/Locales/fr_FR.json:80]: Activer Discord Rich Presence

**Comment by CoolFrenchPerson** [comment]
This text is fine but the alignement is a little weird

image

File: src/Ryujinx.Ava/Assets/Locales/fr_FR.json [hunk 35]
```diff
@@ -427,46 +444,47 @@
...
   "FileLogTooltip": "Sauver le journal de la console dans un fichier journal sur le disque. Cela n'affecte pas les performances.",
-  "StubLogTooltip": "Prints stub log messages in the console. Does not affect performance.",
-  "InfoLogTooltip": "Prints info log messages in the console. Does not affect performance.",
-  "WarnLogTooltip": "Prints warning log messages in the console. Does not affect performance.",
-  "ErrorLogTooltip": "Prints error log messages in the console. Does not affect performance.",
-  "TraceLogTooltip": "Prints trace log messages in the console. Does not affect performance.",
-  "GuestLogTooltip": "Prints guest log messages in the console. Does not affect performance.",
-  "FileAccessLogTooltip": "Prints file access log messages in the console.",
-  "FSAccessLogModeTooltip": "Enables FS access log output to the console. Possible modes are 0-3",
+  "StubLogTooltip": "Affiche les messages de log dans la console. N'affecte pas les performances.",
```
Comment (CoolFrenchPerson) [inline src/Ryujinx.Ava/Assets/Locales/fr_FR.json:477]: In the check box it says stub, but not in the tip that appears when you hover over it, i think this should be changed to "Affiche les messages de journaux stub dans la console. N'affecte pas les performances."

File: src/Ryujinx.Ava/Assets/Locales/fr_FR.json [hunk 36]
```diff
@@ -427,46 +444,47 @@
...
+  "InfoLogTooltip": "Affiche les messages de log d'informations dans la console. N'affecte pas les performances.",
```
Comment (CoolFrenchPerson) [inline src/Ryujinx.Ava/Assets/Locales/fr_FR.json:478]: "Affiche les messages de journaux d'informations dans la console. N'affecte pas les performances."

File: src/Ryujinx.Ava/Assets/Locales/fr_FR.json [hunk 37]
```diff
@@ -427,46 +444,47 @@
...
+  "WarnLogTooltip": "Affiche les messages d'avertissement dans la console. N'affecte pas les performances.",
+  "ErrorLogTooltip": "Affiche les messages de log d'erreur dans la console. N'affecte pas les performances.",
```
Comment (CoolFrenchPerson) [inline src/Ryujinx.Ava/Assets/Locales/fr_FR.json:480]: "Affiche les messages de journaux d'erreur dans la console. N'affecte pas les performances."

File: src/Ryujinx.Ava/Assets/Locales/fr_FR.json [hunk 38]
```diff
@@ -427,46 +444,47 @@
...
+  "TraceLogTooltip": "Affiche la trace des messages de log dans la console. N'affecte pas les performances.",
```
Comment (CoolFrenchPerson) [inline src/Ryujinx.Ava/Assets/Locales/fr_FR.json:481]: "Affiche la trace des messages de journaux dans la console. N'affecte pas les performances.

File: src/Ryujinx.Ava/Assets/Locales/fr_FR.json [hunk 39]
```diff
@@ -427,46 +444,47 @@
...
+  "GuestLogTooltip": "Affiche les messages de log des invités dans la console. N'affecte pas les performances.",
```
Comment (CoolFrenchPerson) [inline src/Ryujinx.Ava/Assets/Locales/fr_FR.json:482]: "Affiche les messages de journaux des invités dans la console. N'affecte pas les performances."

File: src/Ryujinx.Ava/Assets/Locales/fr_FR.json [hunk 40]
```diff
@@ -427,46 +444,47 @@
...
+  "FileAccessLogTooltip": "Affiche les messages de log d'accès aux fichiers dans la console.",
```
Comment (CoolFrenchPerson) [inline src/Ryujinx.Ava/Assets/Locales/fr_FR.json:483]: "Affiche les messages de journaux d'accès aux fichiers dans la console."

**Comment by AcK77** [comment]
All suggestions have to be done on crowdin directly, this PR is handled by Crowdin.

File: src/Ryujinx.Ava/Assets/Locales/fr_FR.json (hunk 33 above)
Comment (AcK77) [reply src/Ryujinx.Ava/Assets/Locales/fr_FR.json:354]: « Appliquette » je suis curieux de voir où on utilise ce mot!

**Comment by Narugakuruga** [comment]
most languages are now 100% completed

**Comment by CoolFrenchPerson** [comment]
French is now back at 100% (it wasnt since 1.1.977)

Also is there a way to translate the 'Unbound' text that appears when there's not key selected in the hotkeys ?

**Pull request merged**